tasks/collect-cosign-params/collect-cosign-params.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-cosign-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task that collects cosign options from the data file
+  params:
+    - name: dataPath
+      type: string
+      description: Path to the JSON string of the merged data containing the configuration options to use
+  workspaces:
+    - name: data
+      description: Workspace to read and save files
+  results:
+    - name: cosign-secret-name
+      type: string
+      description: Name of secret which contains AWS_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and SIGN_KEY
+  steps:
+    - name: collect-params
+      image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
```

Comment:
We currently have a [PR](https://github.com/konflux-ci/release-service-catalog/pull/473/files) open to update all image references to `quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f` . Can you use the same in all your image references as well? That way we don't have to worry about which PR gets merged first and if the other PR needs updating to also cover this.

Reply:
Done.

Reply:
Thanks!